src/world/script_api/push_blocks.c

```diff
@@ -3,7 +3,7 @@
 
 extern PushBlockGrid* wPushBlockGrids[8];
 
-// outVars for values returned by FetchPushedBlockProperties (func_80282E30)
+// outVars for values returned by FetchPushedBlockProperties
 enum {
     BLOCK_PROP_X    = LVar0,
```

Comment:
Ah, I see now these are backwards. X/Y/Z are on LVars 3-5 and I/J/K are on LVars 0-2.